src/Commands/IntakeOn.cpp

```diff
@@ -24,29 +25,29 @@ IntakeIn::IntakeIn(): Command() {
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
 
 // Called just before this Command runs the first time
-void IntakeIn::Initialize() {
+void IntakeOn::Initialize() {
 
 }
 
 // Called repeatedly when this Command is scheduled to run
-void IntakeIn::Execute() {
-	RobotMap::intakeintakeMotor1->Set(0.5);
-	RobotMap::intakeintakeMotor2->Set(0.5);
+void IntakeOn::Execute() {
+	RobotMap::intakeintakeMotor1->Set(-/*speed*/1.0);
+	RobotMap::intakeintakeMotor2->Set(-/*speed*/1.0);
```

Comment:
When tou guys do temporary fixes like this, add a //TODO to remind yourselves to switch it back to using the parameter.